importScripts/sampleRecords/library.destination.json

```diff
@@ -448,9 +671,45 @@
       }
     },
     {
-      "object_id":"14",
-      "id":"14",
-      "idno":"2012\/55",
+      "object_id":"19",
+      "id":"19",
+      "display_label":"Grain pool history",
```

Comment:
This and the next record ("id": "20") are duplicate records in the source database. Previously we were using and `existingRecordPolicy` of `merge_on_idno_with_replace`. The import mapping uses `none`. (see http://docs.collectiveaccess.org/wiki/Data_Importer#Settings for an explaination if you're interested).
This change was made because:
* There are records which have duplicate AccessionNo in them
* Many records do not have an AccessionNo, so CA was merging all records (treating them as identical because the accession number was not actually blank, but `""`.